packages/playground/.flowconfig

```diff
@@ -0,0 +1,75 @@
+[ignore]
```

Comment:
We can ignore needing the flowconfig if we just rely on the built-in App.js. These configs tend to get out of sync with Facebooks. We don't have this hooked into lint anyway it looks like, so this isn't currently CI enforced as far as I can tell.

Reply:
Yeah, I created the initial app using react-native init.  I'll clean up more of the files that we dont need.